styles/_colors.scss

```diff
@@ -0,0 +1,2 @@
+$blue: #1E5CD4;
+$gray: #A9A9A9;
```

Comment:
Don't have any current plans to make this themeable, so going with simple color names for now.

Read through this [article ](https://css-tricks.com/what-do-you-name-color-variables/) for some inspiration, and while semantic names can be a bit more resilient...I do always forget what they actually represent and have to frequently look up what the colors variables represent. 

So for now going with a very simple approach.